src/replication.c

```diff
@@ -42,13 +42,21 @@
 #include <stdatomic.h>
 #include <sys/socket.h>
 #include <sys/stat.h>
+#include <ctype.h>
 
 void replicationDiscardCachedPrimary(void);
 void replicationResurrectCachedPrimary(connection *conn);
+void replicationResurrectProvisionalPrimary(void);
 void replicationSendAck(void);
 int replicaPutOnline(client *replica);
 void replicaStartCommandStream(client *replica);
 int cancelReplicationHandshake(int reconnect);
+void syncWithPrimary(connection *conn);
```

Comment:
I think we should start using `static` more (and I would also like to see us converging on using `_` prefix for internal only functions but that can wait for a consensus with @valkey-io/core-team)
```suggestion
static void syncWithPrimary(connection *conn);
```

Reply:
👍